### ISSUE TITLE
UI Spec for Network Attribute Tools v 1.0

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/riverscapes-tools#392]
- Finish the metric display on left
- How does Network Attributes integrate into Riverscapes Studio (web)?

**Comment by joewheaton** [comment]
@philipbaileynar once we finish up the QRiS scoping estimate, I'd like to update this thread here.

**Comment by philipbaileynar** [comment]
Yes, but this needs to get moved to QRiS repo where the UI will reside.
